source/adios2/engine/bp5/BP5Writer.cpp

```diff
@@ -860,8 +860,7 @@ uint64_t BP5Writer::CountStepsInMetadataIndex(format::BufferSTL &bufferSTL)
 
         switch (recordID)
         {
-        case IndexRecord::WriterMapRecord:
-        {
```

Comment:
Wait, this is just a formatting change...  Is format checking still broken?

Reply:
I assume that is clang-format 16 vs 7, but it is accepted. The column limit in release_29 is still 80 characters and this is still the case here.

Reply:
Ah, I forgot this was 2.9.  Format checking may still be broken if that fix came with the 100 character limit and so was only merged with master.